lib/dbservice/profile/student_profile.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Dbservice.Profiles.StudentProfile do
+  @moduledoc false
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Dbservice.Profiles.UserProfile
+  alias Dbservice.Users.Student
+
+  schema "student_profile" do
+    field(:student_id, :string)
+    field(:took_test_atleast_once, :boolean)
+    field(:took_class_atleast_once, :boolean)
+    field(:total_number_of_tests, :integer)
+    field(:total_number_of_live_classes, :integer)
+    field(:attendance_in_classes_current_year, {:array, :decimal})
+    field(:classes_activity_cohort, :string)
+    field(:attendance_in_tests_current_year, {:array, :decimal})
+    field(:tests_activity_cohort, :string)
+    field(:performance_trend_in_fst, :string)
+    field(:max_batch_score_in_latest_test, :integer)
+    field(:average_batch_score_in_latest_test, :decimal)
+    field(:tests_number_of_correct_questions, :integer)
+    field(:tests_number_of_wrong_questions, :integer)
+    field(:tests_number_of_skipped_questions, :integer)
+    # add plio data later
+
+    timestamps()
+
+    belongs_to(:user_profile, UserProfile)
+    belongs_to(:student, Student, foreign_key: :student_fk)
```

Comment:
Just wanted to confirm, we have a migration for `student_profile` having field name `student_fk_id`. But here we are establishing association using `student_fk`. Is this correct?

Reply:
there's no student_fk_id.. there's just student_fk. earlier it was student_fkey_id, i renamed it for consistency. deleted and recreated index too